### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: tests/test_basics.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from importlib import resources
```
Comment (bluca) [inline tests/test_basics.py:1]: license line

**Comment by bluca** [comment]
we should get GH actions enabled

**Comment by mrc0mmand** [comment]
> we should get GH actions enabled

*Did someone summon me?*

**Comment by bluca** [comment]
_There was a disturbance in the Force_

**Comment by mrc0mmand** [comment]
(I'll look into it once this is merged, since it requires a branch directly in the "upstream" repo.)

**Comment by bluca** [comment]
thanks!

**Comment by keszybz** [comment]
Updated with the license line and some more gitattributes. I guess this can be merge dnow.

**Review by bluca — APPROVED** [review]

**Pull request merged**